### ISSUE TITLE
Twitter Kit

### ISSUE THREAD

**Issue by annamf** [mano-mykingdom/crashlytics#6]
Hi, 

Is included in this module the Twitter Kit ?? Is possible to use that functions??

Thank you.

**Comment by manojdcoder** [comment]
No, didn't get time for them. Happy to see a pull request if you can.
